packages/calcite-components/src/components/tab-nav/tab-nav.tsx

```diff
@@ -414,9 +673,38 @@ export class TabNav {
     }
 
     requestAnimationFrame(() => {
-      this.updateOffsetPosition();
-      this.updateActiveWidth();
+      this.updateActiveIndicator();
       tabTitles[this.selectedTabId].focus();
     });
   }
+
+  private renderScrollButton = (overflowDirection: "start" | "end"): VNode => {
+    const { bordered, messages, hasOverflowingStartTabTitle, hasOverflowingEndTabTitle, scale } =
+      this;
+    const isEnd = overflowDirection === "end";
+
+    return (
+      <div
+        class={{
+          [CSS.scrollButtonContainer]: true,
+          [CSS.scrollBackwardContainerButton]: !isEnd,
+          [CSS.scrollForwardContainerButton]: isEnd,
+        }}
+        hidden={(isEnd && !hasOverflowingEndTabTitle) || (!isEnd && !hasOverflowingStartTabTitle)}
```

Comment:
Instead of hiding, should it just not be rendered? Not a deal breaker just wondering.